mariner-installer/build/build

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+echo "BRANCH: $BRANCH"
+cd $WORKSPACE/mariner-installer
+
+. /etc/os-release
```

Comment:
The reason for sourcing `/etc/os-release` is so that `"$ID" == 'ubuntu'`

We've got a couple different OS detection mechanisms lying around in this tree, and we should probably standardize at some point